lib/studies/reply/mail_view_page.dart

```diff
@@ -1,14 +1,98 @@
 import 'package:flutter/material.dart';
+import 'package:gallery/studies/reply/colors.dart';
 
 class MailViewPage extends StatelessWidget {
   const MailViewPage({Key key}) : super(key: key);
   @override
   Widget build(BuildContext context) {
+    final textTheme = Theme.of(context).textTheme;
+
     return Scaffold(
-      body: Center(
-        child: OutlineButton(
-          onPressed: () => Navigator.pop(context),
-          child: const Text('Back'),
+      body: SafeArea(
+        child: Padding(
+          padding:
+              const EdgeInsetsDirectional.only(top: 42, start: 20, end: 20),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Row(
+                children: [
+                  Expanded(
+                    child: Text(
+                      'Package shipped!',
+                      style: textTheme.headline4.copyWith(height: 1.1),
+                    ),
+                  ),
+                  IconButton(
+                    icon: const Icon(Icons.keyboard_arrow_down),
+                    onPressed: () {
+                      Navigator.pop(context);
+                    },
+                    splashRadius: 20,
+                  ),
+                ],
+              ),
+              const SizedBox(height: 16),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: [
+                  Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    mainAxisSize: MainAxisSize.min,
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: [
+                      Row(
+                        children: [
+                          const Text('Google Express'),
+                          Padding(
+                            padding: const EdgeInsetsDirectional.only(
+                              start: 8,
+                              end: 8,
+                            ),
+                            child: Container(
+                              height: 1,
+                              width: 8,
+                              color: Colors.black,
+                            ),
+                          ),
```

Comment:
I think you should just use an em-dash for this instead of a container. Then you can remove the parent row and just have:

`Text('Google Express — 4 hrs ago')`

Reply:
I'll update this.